webapp/src/main/webapp/themes/cu-boulder/templates/publications.ftl

```diff
@@ -52,22 +52,23 @@
                 render_result_record: function(options, record)
                 {
 
-                    var doi = record["doi"];
+                    var pub = record["uri"];
+		    var doi = record["doi"];
                     var doiUrl = "https://dx.doi.org/"+record["doi"];
                     var escapedDOI = encodeURIComponent(doi).replace(/-/g, "--");
                     var scholarBadgeURL = "https://img.shields.io/badge/Open_Access-CU_Scholar-orange.svg?style=social&logo=open-access&logoColor=orange";
                     var doiBadgeURL = "https://img.shields.io/badge/DOI-" + escapedDOI + "-blue.svg?style=social&labelColor=black";
                     var html = "<tr><td>";
 
                     if (record["name"]) {
-                        html += "<strong><h4><a href=\""+record["uri"]+"\" target=\"_blank\">"+record["name"]+"</a></h4></strong>";
+			html += "<strong><h4><a href=\""+record["uri"]+"\" target=\"_blank\" onclick=\"ga('send', 'event', 'Experts Link', this.href, 'Publication Label'); return true;\" >"+record["name"]+"</a></h4></strong>";
```

Comment:
Does this name 'Experts Link' give enough detail? (Same for the authors, subject areas and publications links below.)

Reply:
did you have a chance to review the data in google analytics? The this.href variable provides the actual target link to GA. It looks good from my perspective there